doc/devbox_setup.md

```diff
@@ -336,6 +336,12 @@ When project generation completes you will see an XCode project file (.xcodeproj
 the `cmake` folder. To build the SDK, open **cmake\azure_iot_sdks.xcodeproj** in XCode and 
 use XCode's build and run features.
 
+**Note:** Until Mac updates the Curl library to version to 7.58 or greater it will also be necessary
+to modify your XCode project file (.xcodeproj) to replace all the lines that read</br>
+`LIBRARY_SEARCH_PATHS = "";`</br>
+with</br>
+`LIBRARY_SEARCH_PATHS = "/usr/local/Cellar/curl/7.58.0/lib";`</br>
+
```

Comment:
Please add (copy the text below as markup):

The example above assumes curl 7.58 has been compiled and saved into `/usr/local/Cellar/curl/7.58.0`. For more details please see "[Upgrade CURL on Mac OS](#Upgrade-CURL-on-Mac-OS)".